src/pages/ThorChainLP/components/ReusableLpConfirm.tsx

```diff
@@ -191,8 +192,17 @@ export const ReusableLpConfirm: React.FC<ReusableLpConfirmProps> = ({
   const confirmCopy = useMemo(() => {
     if (isTradingActive === false) return translate('common.poolHalted')
 
-    return translate('pools.confirmAndDeposit')
-  }, [isTradingActive, translate])
+    const message = (() => {
+      if ('feeAmountFiat' in confirmedQuote) {
+        // 'feeAmountFiat' exists, so confirmedQuote is an instance of LpConfirmedDepositQuote
```

Comment:
Wondering if we want to use these heuristics as a discriminator vs. keeping `feeAmountFiat` (i.e 0) for the withdraw quote and use a deposit/withdraw terminology in fields, e.g `assetCryptoLiquidityAmount` -> `assetCryptoDepositAmount` / `assetCryptoWithdrawAmount` ?

Reply:
That makes way more sense 🧠 will do so in a follow-up PR!

Reply:
Actioned here: https://github.com/shapeshift/web/pull/6277